src/dispatch/plugins/dispatch_core/config.py

```diff
@@ -1,3 +1,6 @@
 from starlette.config import Config
 
 config = Config(".env")
+
+DISPATCH_JWT_AUDIENCE = config("DISPATCH_JWT_AUDIENCE")
```

Comment:
Can these be gated by if the plugin PCKE plugin is actually in use? See [here](https://github.com/Netflix/dispatch/blob/develop/src/dispatch/config.py#L85) for an example. 

We could also probably move those values to the dispatch_core config as well... But I can tackle that in a separate PR.